documentation/src/docs/table-api.md

```diff
@@ -344,4 +351,52 @@ deployment:
   # (optional) port from cluster to gateway
   gateway-port: 0
 
-```
\ No newline at end of file
+```
+
+### Sample DDL
+
+Here is a sample DDL for reference which can be used as a source as well as sink to read from and write data into Pravega as table records.
+
+But the stream `streamX` should be created in advance to have it working.
+
+```
+CREATE TABLE sample (
+  category STRING,
+  cnt INT,
+  tp TIMESTAMP(3)

```

Comment:
`ts` is a better name. It worths to introduce the watermark in DDL `WATERMARK FOR ts as ts - INTERVAL '5' SECOND` here. (ref. https://cwiki.apache.org/confluence/display/FLINK/FLIP-66%3A+Support+Time+Attribute+in+SQL+DDL)